app/views/shared/_header.html.erb

```diff
@@ -1,25 +1,20 @@
-<nav class="bg-white">
-  <div id ='nav' class="flex items-center justify-between content-end flex-wrap bg-white p-5 shadow-lg">
+<nav>
+  <div id ='nav' class="navbar">
 <!-- Menu hamburguesa-->
-    <div class="block">
-        <button id='menu' class="flex items-center px-3 py-2">
-        <svg class="fill-current h-4 w-4" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><title>Menu</title><path d="M0 3h20v2H0V3zm0 6h20v2H0V9zm0 6h20v2H0v-2z"/></svg>
-        </button>
+    <div class="navbar-hamburguer navbar-mr">
+      <button id='menu'>
+        <i class="fas fa-bars"></i>
+      </button>
```

Comment:
The hamburger menu does not appear.

![imagen](https://user-images.githubusercontent.com/12243331/104506980-ba66cd00-55ab-11eb-8be2-5c38e7bb4e5e.png)

Reply:
Is this running on the branch of this PR?
If so, what its the viewport you are using?

Reply:
You could also check if you installed the yarn packages added in this issue (just Font Awesome if I'm not wrong), maybe that's the reason you are not seeing any hamburger menu, because I changed its icon to use Font Awesome

Reply:
Verified.